selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/DockerUtil.java

```diff
@@ -62,14 +62,31 @@ public boolean isCheRunLocally() {
   /**
    * Copies files container to the host computer.
    *
-   * @param containerId ID of container which we is coping from
-   * @param pathInsideContainer destination of files to copy
-   * @param copyTo placement of files on the host computer
+   * @param containerId ID of container which we is being copped from
+   * @param pathInsideContainer destination of files in the container
+   * @param pathOnHost placement of files on the host computer
    * @throws IOException
    */
-  public void copy(String containerId, Path pathInsideContainer, Path copyTo) throws IOException {
+  public void copyFromContainer(String containerId, Path pathInsideContainer, Path pathOnHost)
+      throws IOException {
     String copyCommand =
-        format("docker cp %1$s:%2$s %3$s", containerId, pathInsideContainer, copyTo);
+        format("docker cp %1$s:%2$s %3$s", containerId, pathInsideContainer, pathOnHost);
```

Comment:
what about some additional checking:
execute **```docker ps -aq | grep <containerId>```**
and check that response not empty.
It gives as to ensure the container existence

Reply:
The paths also may be checked for existence.

Reply:
It's actually responsibility of client, who uses the method, to provide correct id and path. Client will end up with exception in case of incorrect id or path.